javascriptv3/example_code/codecommit/README.md

```diff
@@ -40,7 +40,7 @@ node [example name].js
 ```
 
 ## Unit tests
-For more information see, the [README](../README.rst).
+For more information see, the [README](../../README.md#tests).
```

Comment:
Is this a Writeme generated file? It looks different than the rest.

Reply:
CodeCommit isn't part of our metadata yet.